dotnet/src/webdriver/DevTools/Json/JsonEnumMemberConverter.cs

```diff
@@ -1,25 +1,30 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using System.Runtime.Serialization;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
 namespace OpenQA.Selenium.DevTools.Json
 {
-    internal class JsonEnumMemberConverter<TEnum> : JsonConverter<TEnum> where TEnum : Enum
+    internal class JsonEnumMemberConverter<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] TEnum>
+        : JsonConverter<TEnum> where TEnum : struct, Enum
     {
         private readonly Dictionary<TEnum, string> _enumToString = new Dictionary<TEnum, string>();
         private readonly Dictionary<string, TEnum> _stringToEnum = new Dictionary<string, TEnum>();
 
         public JsonEnumMemberConverter()
         {
             var type = typeof(TEnum);
-            var values = Enum.GetValues(type);
-
+#if NET5_0_OR_GREATER
+            TEnum[] values = Enum.GetValues<TEnum>();
+#else
+            Array values = Enum.GetValues(type);
+#endif
             foreach (var value in values)
```

Comment:
`var` here is actually `TEnum` in .NET, but `object` in .NET Standard. All use of `value` is either cast or `ToString()`-ed, so there's no runtime difference.